staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -162,8 +162,13 @@ func createHandler(r rest.NamedCreater, scope *RequestScope, admit admission.Int
 		userInfo, _ := request.UserFrom(ctx)
 
 		if objectMeta, err := meta.Accessor(obj); err == nil {
-			// Wipe fields which cannot take user-provided values
-			rest.WipeObjectMetaSystemFields(objectMeta)
+			preserveObjectMetaSystemFields := false
+			if c, ok := r.(rest.SubresourceObjectMetaPreserver); ok && len(scope.Subresource) > 0 {
+				preserveObjectMetaSystemFields = c.PreserveRequestObjectMetaSystemFieldsOnSubresourceCreate()
+			}
+			if !preserveObjectMetaSystemFields {
```

Comment:
This doesn't verify that the input was valid, it just chooses not to ignore it.  Do we validate somewhere else that it was correct?

Reply:
none of our subresource creates actually persist new objects, the only place this is being allowed, the uid on the input is used as a precondition to match the existing parent resource, so yes, it has to be valid